src/index.ts

```diff
@@ -0,0 +1,153 @@
+import escape from 'xml-escape';
+import * as Cypress from 'cypress';
+
+// This is necessary because Eslint is unaware of the global namespace provided by Cypress.
+// By declaring it within this project, we inform Eslint about its existence.
+/* global CypressCommandLine:readonly */
```

Comment:
Imho it'd be better to declare this global in the `.eslintrc.js` file, as it's closer to the where the rules are defined.